mmv1/third_party/terraform/services/netapp/resource_netapp_volume_test.go

```diff
@@ -659,3 +658,60 @@ func testAccNetappVolume_volumeBasicExample_cleanupScheduledBackup(t *testing.T,
 		return nil
 	}
 }
+
+func TestAccNetappVolume_autoTieredNetappVolumeExample_update(t *testing.T) {
```

Comment:
```suggestion
func TestAccNetappVolume_autoTieredNetappVolume(t *testing.T) {
```

Reply:
Applying this since we're not doing an update / this is a handwritten test rather than an example-based one.